vllm/attention/ops/triton_unified_attention.py

```diff
@@ -819,7 +831,7 @@ def _get_tile_size(
         return 32
 
     # Default behavior
-    if is_prefill:
+    if is_prefill or use_2d_kernel:
```

Comment:
Can you explain a bit more about why we are changing it to always use tile size 32 when using the 2D kernel?